mne/viz/topomap.py

```diff
@@ -145,13 +145,37 @@ def _add_colorbar(ax, im, cmap, side="right", pad=.05, title=None,
     divider = make_axes_locatable(ax)
     cax = divider.append_axes(side, size=size, pad=pad)
     cbar = plt.colorbar(im, cax=cax, cmap=cmap, format=format)
-    if cmap[1]:
+    if cmap is not None and cmap[1]:
         ax.CB = DraggableColorbar(cbar, im)
     if title is not None:
         cax.set_title(title, y=1.05, fontsize=10)
     return cbar, cax
 
 
+def _remove_zeros(proj):
+    """Remove grad or mag data if only contains 0s (gh 5641)."""
+    import re
+    grad_regex = '^MEG.*[23]$'
+    meg_regex = '^MEG.*1$'
```

Comment:
I do not like very much the use of regular expressions when the task is easy to accomplish with list comprehensions. Most of our devs do not think in regexp.
How about a stupid but effective ```.endswith('1')``` ?